datasources.go

```diff
@@ -36,6 +42,16 @@ func (api API) CreateDataSource(name string) (*DataSource, error) {
 	return ds, err
 }
 
+// CreateDataSource2 creates an API Data Source in ChartMogul.
+// * Allows other parameters than just the name.
+//
+// See https://dev.chartmogul.com/v1.0/reference#data-sources
+func (api API) CreateDataSource2(dataSource *DataSource) (*DataSource, error) {
```

Comment:
@quimrstorres interesting idea, but unfortunately those are methods defined on a struct, can't mix packages there. It has to be a different name. Any ideas? :)
Side note: Go doesn't have a good deprecation mechanism, but anyway *we don't need to deprecate the original one*. `System=` is just for our advanced integrations.